net/http.ts

```diff
@@ -323,3 +324,13 @@ async function readAllIterator(
   }
   return collected;
 }
+
+const http = {
+  Handle: function() {
+
+  },
+  FileServer,
```

Comment:
I prefer naming this `fileServer`. Also, it feels more natural to me to actually move all the fileServer code to `http.ts`.

`file_server.ts` would look similar to somthing like this after the move:
```ts
import { cwd, args } from "deno";
import { fileServer } from "./http.ts";

const serverArgs = args.slice();
let CORSEnabled = false;
for (let i = 0; i < serverArgs.length; i++) {
  if (serverArgs[i] === "--cors") {
    CORSEnabled = true;
    serverArgs.splice(i, 1);
    break;
  }
}
let currentDir = cwd();
const target = serverArgs[1];
if (target) {
  currentDir = `${currentDir}/${target}`;
}
const addr = `0.0.0.0:${serverArgs[2] || 4500}`;

fileServer(currentDir, addr, {
  cors: CORSEnabled,
})
```